apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx

```diff
@@ -129,6 +134,15 @@ const SetupAnalyticsModal = ({
 						label="API key"
 						name="apiKey"
 						placeholder="No API key"
+						{...(isKeyGenerated
+							? {
+									leftIcon: (
+										<div className={styles["generated-key-indicator"]}>
+											<Icon height={20} name="check" width={20} />
+										</div>
+									),
+								}
+							: {})}
```

Comment:
@liza-veis I'm not sure about this
```
									leftIcon: {
										isKeyGenerated ? <div className={styles["generated-key-indicator"]}>
											<Icon height={20} name="check" width={20} />
										</div> : <></> (or undefined)
									},
```
The code above looks better, but it doesn't work properly. So it's a solution, but I don't like the way the code looks.

Reply:
we can use && here instead of ?, this will simplify it a bit

Reply:
@liza-veis I tried, but...
![image](https://github.com/user-attachments/assets/6f62d3c3-0e02-40a6-84dc-55d48fa92dd5)

Reply:
in this case, let's leave it as it is